src/klein/test/test_form.py

```diff
@@ -365,7 +373,22 @@ def test_noName(self) -> None:
         errors = self.flushLoggedErrors(ValueError)
         self.assertEqual(len(errors), 1)
         self.assertIn(
-            str(errors[0].value), "Cannot extract unnamed form field."
+            "Cannot extract unnamed form field.",
+            str(errors[0].value),
+        )
+
+        self.successResultOf(
```

Comment:
the docstring for `test_noName` is quite brief...
I am not sure if this new code is in the same scope, or if this requires a separate test method, with a separate docstring.

Reply:
(Same issue as above, so handling the same way).